webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/db/Doobie.scala

```diff
@@ -172,12 +172,17 @@ object Doobie {
   )
 
   implicit val ReportRead: Read[Reports] = {
-    type R = (DateTime, RuleId, DirectiveId, NodeId, Int, String, String, DateTime, String, String)
+    type R = (DateTime, RuleId, String, NodeId, Int, String, String, DateTime, String, String)
```

Comment:
Why not DirectiveRId here ?

Reply:
to be able to parse the string as a directiverid. We could have a separate `Read[DirectiveRId]` but I wanted to limit changes for a first go.